### PR TITLE
Rename the environment variable FRONTEND_URL to CORS_ALLOWED_ORIGIN_LIST and accept list

### LINKED ISSUE #156: Rename the environment variable `FRONTEND_URL` to `CORS_ALLOWED_ORIGIN_LIST`
To be more general meaning.
https://github.com/kubernetes-sigs/kube-scheduler-simulator/blob/master/config/config.go#L25

/kind cleanup

### PR DESCRIPTION
**Pull request by sanposhiho**
#### What type of PR is this?

/kind cleanup
/kind feature

#### What this PR does / why we need it:

Rename the environment variable FRONTEND_URL to CORS_ALLOWED_ORIGIN_LIST and accept list.

#### Which issue(s) this PR fixes:

Fixes #156 

#### Special notes for your reviewer:

/label tide/merge-method-squash

### CODE REVIEW

File: config/config.go [hunk 1]
```diff
@@ -111,13 +112,28 @@ func getEtcdURL() (string, error) {
...
+// getCorsAllowedOriginList fetches CorsAllowedOriginList from the env named CORS_ALLOWED_ORIGIN_LIST.
+// This allowed list is applied to kube-apiserver and the simulator server.
+//
+// e.g. CORS_ALLOWED_ORIGIN_LIST="http://localhost:3000, http://localhost:3001, http://localhost:3002"
+//       → return []string{"http://localhost:3000", "http://localhost:3001", "http://localhost:3002"}
+func getCorsAllowedOriginList() ([]string, error) {
+	e := os.Getenv("CORS_ALLOWED_ORIGIN_LIST")
 	if e == "" {
-		return "", xerrors.Errorf("get FRONTEND_URL from env: %w", ErrEmptyEnv)
+		return nil, xerrors.Errorf("get CORS_ALLOWED_ORIGIN_LIST from env: %w", ErrEmptyEnv)
```
Comment (196Ikuchil) [inline config/config.go:123]: Should I also raise an error if the URL is informal? (Or fix to return `""`?)

**Review by 196Ikuchil — COMMENTED** [review]
Please check it✋

File: config/config.go (hunk 1 above)
Comment (sanposhiho) [reply config/config.go:123]: Updated.

**Comment by 196Ikuchil** [comment]
Thanks!
/lgtm

**Pull request merged**